Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -174,9 +182,6 @@ private extension TabScrollingController {
 
         var updatedOffset = headerTopOffset - delta
         headerTopOffset = clamp(updatedOffset, min: -topScrollHeight, max: 0)
-        if isHeaderDisplayedForGivenOffset(updatedOffset) {
```

Comment:
Nit: Just noticed that `isHeaderDisplayedForGivenOffset` isn't used anymore after this deletion. Want to remove the method as well?